test/suite/db_clean.go

```diff
@@ -1,4 +1,4 @@
-package cleaner
+package suite
```

Comment:
Let's also replace db_clean.go by the code we have in Auth. We improved it in Auth too :)
The current (WIT) implementation supports cleaning up records with the only primary key. The advanced version from But Auth also supports composite primary keys (multiple-keys) - https://github.com/fabric8-services/fabric8-auth/blob/master/gormsupport/cleaner/db_clean.go
We use tables with composite primary keys in Auth DB for example.